packages/http/src/decorators.ts

```diff
@@ -456,11 +456,10 @@ export const $server: ServerDecorator = (
     servers = [];
     context.program.stateMap(HttpStateKeys.servers).set(target, servers);
   }
-  servers.push({
-    url,
-    description,
-    parameters: parameterMap,
-  });
+  const server: HttpServer = description
```

Comment:
you can still do what you did before, if a property is undefined its the same 

 servers.push({
    url,
    description,
    parameters: parameterMap,
  });